src/OrleansVSTools/VSProjectTemplateGrainImplementation/ProjectTemplate.csproj

```diff
@@ -46,7 +46,6 @@
     <Compile Include="Properties\orleans.codegen.cs" />
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
-  <Import Project="$(SolutionDir)packages\Microsoft.Orleans.OrleansCodeGenerator.Build.1.2.0\build\Microsoft.Orleans.OrleansCodeGenerator.Build.targets" Condition="Exists('$(SolutionDir)packages\Microsoft.Orleans.OrleansCodeGenerator.Build.1.2.0\build\Microsoft.Orleans.OrleansCodeGenerator.Build.targets')" />
```

Comment:
so these will be added automatically when the template unfolds and installs OrleansCodeGenerator.Build?